docs/content/en/docs/reference/clusterspec/vsphere.md

```diff
@@ -80,20 +80,21 @@ kind: VSphereMachineConfig
-  storagePolicyName: <span style="color:red">"storagePolicy1"</span>    <a href="#storagepolicyname-optional"># Storage policy name associated with VMs</a>
-  template: <span style="color:red">"bottlerocket-kube-v1-25"</span>    <a href="#template-optional"># VM template for EKS Anywhere (required for RHEL/Ubuntu-based OVAs)</a>
-  users:                             <a href="#users-optional"># Add users to access VMs via SSH</a>
-  - name: <span style="color:red">"ec2-user"</span>                 <a href="#users0name-optional"># Name of each user set to access VMs</a>
-    sshAuthorizedKeys:               <a href="#users0sshauthorizedkeys-optional"># SSH keys for user needed to access VMs</a>
+  diskGiB:  <span style="color:green">25</span>                         <a href="#diskgib-optional"># Size of disk on VMs, if no snapshots</a>
+  datastore: <span style="color:red">"datastore1"</span>              <a href="#datastore-required"># Path to vSphere datastore to deploy EKS Anywhere on (required)</a>
+  folder: <span style="color:red">"folder1"</span>                    <a href="#folder-required"># Path to VM folder for EKS Anywhere cluster VMs (required)</a>
+  numCPUs: <span style="color:green">2</span>                           <a href="#numcpus-optional"># Number of CPUs on virtual machines</a>
+  memoryMiB: <span style="color:green">8192</span>                      <a href="#memorymib-optional"># Size of RAM on VMs</a>
+  osFamily: <span style="color:red">"bottlerocket"</span>             <a href="#osfamily-optional"># Operating system on VMs</a>
+  resourcePool: <span style="color:red">"resourcePool1"</span>        <a href="#resourcepool-required"># vSphere resource pool for EKS Anywhere VMs (required)</a>
+  storagePolicyName: <span style="color:red">"storagePolicy1"</span>  <a href="#storagepolicyname-optional"># Storage policy name associated with VMs</a>
+  template: <span style="color:red">"bottlerocket-kube-v1-25"</span>  <a href="#template-optional"># VM template for EKS Anywhere (required for RHEL/Ubuntu-based OVAs)</a>
+  cloneMode: <span style="color:red">"fullClone"</span>               <a href="#clonemode-optional"># Clone mode to use when cloning VMs from the template</a>
+  users:                               <a href="#users-optional"># Add users to access VMs via SSH</a>
+  - name: <span style="color:red">"ec2-user"</span>                   <a href="#users0name-optional"># Name of each user set to access VMs</a>
+    sshAuthorizedKeys:                 <a href="#users0sshauthorizedkeys-optional"># SSH keys for user needed to access VMs</a>
     - <span style="color:red">"ssh-rsa AAAAB3NzaC1yc2E..."</span>
-  tags:                              <a href="#tags-optional"># List of tags to attach to cluster VMs, in URN format</a>
+  tags:                                <a href="#tags-optional"># List of tags to attach to cluster VMs, in URN format</a>
```

Comment:
Most of the changes here are to fix the indentation for the comments. Here's how it looks rendered
<img width="967" alt="image" src="https://user-images.githubusercontent.com/12872943/230157938-5a63f7b2-42cd-4747-bbc8-bec842adb1fe.png">

Reply:
Sure. That's fine with me.